### DISCUSSION TITLE
[ZK Stack] - Native relayer/paymaster without AA and settling on different chain (e.g., Base)

### DISCUSSION THREAD

**Discussion by MikelinoZk** [zkSync-Community-Hub/zksync-developers#498] [ZK Stack] [answered]
### Module Affected

- [ ] System contracts (era-system-contracts)
- [ ] Smart contracts (era-contracts)
- [ ] zkEVM (era-zk_evm)
- [ ] Server (zksync-era)
- [ ] Prover (era-boojum)
- [ ] Fee withdrawer (era-fee-withdrawer)
- [ ] Block Explorer (clock-explorer)
- [ ] Portal - Wallet + Bridge (dapp-portal)
- [X] Other (please specify below)

### Rust Version

1.75.0

### Operating System

- [ ] Ubuntu
- [ ] MacOS
- [ ] Windows
- [X] Other (Please specify below)

### Issue Description

- Do you know if there is a native relayer/paymaster without AA (like celo, more of this here (https://docs.celo.org/protocol/transaction/erc20-transaction-fees))
 - Do customers have a way to settle on another L2 like Base (e.g., [DEGEN Launches Arbitrum Orbit-Based Layer 3 Chain - The Defiant] (https://thedefiant.io/news/defi/degen-launches-arbitrum-orbit-based-l3-leveraging-base))

### Expected Behavior

N/A - more of a broader question

### Current Behavior

N/A - more of a broader question

### Prior Research

- [X] Yes, I searched for similar issues and did not find any matching ones.

**Comment by bxpana** [comment]
Appreciate the post, will take some time to review and get back to you

**Reply by bxpana** [reply, marked as answer]
> Do you know if there is a native relayer/paymaster without AA (like celo, more of this here (https://docs.celo.org/protocol/transaction/erc20-transaction-fees))

For ZK Stack yes we plan to have this as an option and currently working on being able to change the base token for the ZK Stack chain you deploy, https://x.com/zkSyncDevs/status/1775921160640761907

> Do customers have a way to settle on another L2 like Base (e.g., [DEGEN Launches Arbitrum Orbit-Based Layer 3 Chain - The Defiant] (https://thedefiant.io/news/defi/degen-launches-arbitrum-orbit-based-l3-leveraging-base))

Yes this will also be available and being worked on, https://x.com/zkSyncDevs/status/1775921157750895033

You can follow the Dev account on Twitter to keep up with the latest updates https://twitter.com/zkSyncDevs
